docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1953,6 +1953,7 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/core/capabilityStatementCacheTimeout`|integer|The number of minutes that a tenant's CapabilityStatement is cached for the metadata endpoint. |
 |`fhirServer/core/extendedCodeableConceptValidation`|boolean|A boolean flag which indicates whether extended validation is performed by the server during object construction for code, Coding, CodeableConcept, Quantity, Uri, and String elements which have required bindings to value sets.|
 |`fhirServer/core/disabledOperations`|string|A comma-separated list of operations which are not allowed to run on the IBM FHIR Server, for example, `validate,import`. Note, do not include the dollar sign `$`|
+|`fhirServer/core/defaultPageSize`|integer|Sets the pageSize to use in search when no _count parameter is specified in the request. If a user-specified value exceeds the max page size (1000), then a warning is logged and max page size will be used. If not provided, the default page size (10) is used.|
```

Comment:
I assume it applies to history interactions as well
```suggestion
|`fhirServer/core/defaultPageSize`|integer|Sets the pageSize to use in search and history interactions when no _count parameter is specified in the request. If a user-specified value exceeds the max page size (1000), then a warning is logged and max page size will be used. If not provided, the default page size (10) is used.|
```

Reply:
Also, its a bit confusing, but we have 3 separate tables in section 5.1.   You added this property to the table that has the description.  However, 5.1.2 has the default value and 5.1.3 indicates whether the parameter is tenant-specific (different values allowed for different tenants) and dynamic (changes take affect without a server restart).
We used to have all of this in a single table, but it was too wide for just about any documentation format.
For 5.1.2, just add a row and indicate the default is 10.
For 5.1.3, since you're using FHIRConfigHelper, this property will be dynamic and tenant-specific, so thats just Y|Y.